src/playout/mod.rs

```diff
@@ -74,21 +75,23 @@ pub trait Playout: Sync + Send {
         size as usize * size as usize * 3
     }
 
-    fn select_move(&self, board: &Board) -> Move {
+    fn select_move(&self, board: &Board, rng: &mut XorShiftRng) -> Move {
         let color = board.next_player();
         let vacant = board.vacant();
-        let playable_move_exists =  vacant
+        let playable_move =  vacant
```

Comment:
it's a tiny speedup when your first vacant squares are all eyes or illegal, since we already checked them

these two changes are about 1% faster which is nothing to write home about, but I thought of them already so they're free